src/country_workspace/contrib/hope/constants.py

```diff
@@ -14,6 +14,67 @@
 ACCOUNT_FIELDSET_NAME: Final[str] = "HOPE Account"
 DOCUMENT_FIELDSET_NAME: Final[str] = "HOPE Document"
 
-DOCUMENT_TYPES = ("national_id", "national_passport")
+# Updated document types to match HOPE Core API
+DOCUMENT_TYPES = ("national_id", "national_passport", "birth_certificate", "other")
 
-ACCOUNT_TYPES = ("mobile", "bank")
+# Updated account types to match HOPE Core API
+ACCOUNT_TYPES = ("mobile", "bank", "cash", "other")
+
+# Field mappings for HOPE Core API transformation
+INDIVIDUAL_FIELD_MAPPINGS = {
+    "given_name": "first_name",
```

Comment:
the whole concept of country workspace is to avoid having these mappings. We have flex field library to address this! @patsatsia